### PR TITLE
Add section to guide about setting up ESLint and Prettier

### PR DESCRIPTION
**Pull request by peterlazar1993**

### CODE REVIEW

**Comment by peterlazar1993** [comment]
@brentvatne Let me know if I should anything more to this commit

File: react-native-scripts/template/README.md [hunk 1]
```diff
@@ -142,6 +143,34 @@ You can optionally use a [plugin for your IDE or editor](https://flow.org/en/doc
...
+
+```json
+{
+  "extends": ["react-native"]
+}
+```
+
+Please see the repo of [eslint-config-react-native](https://github.com/kunalgolani/eslint-config/tree/master/react-native) to know what rules are enabled with this configuration.
+
+After configuring eslint, run `npm run lint` (or `yarn lint`) to check for linting erros.
```
Comment (brentvatne) [inline react-native-scripts/template/README.md]: linking erros -> linting errors

File: react-native-scripts/template/README.md [hunk 2]
```diff
@@ -142,6 +143,34 @@ You can optionally use a [plugin for your IDE or editor](https://flow.org/en/doc
 
 To learn more about Flow, check out [its documentation](https://flow.org/).
 
+## Adding ESLint and Prettier
+
+[ESLint](http://eslint.org/) is a configurable JavaScript linter. ESLint can help standardize a style guide across the team.
+
+To add eslint to a Create React Native App project, follow these steps:
```
Comment (brentvatne) [inline react-native-scripts/template/README.md]: eslint should be ESLint here -- make sure to use casing consistently when using outside of context of executing command :) I might have missed somewhere else

File: react-native-scripts/template/README.md [hunk 3]
```diff
@@ -142,6 +143,34 @@ You can optionally use a [plugin for your IDE or editor](https://flow.org/en/doc
 
 To learn more about Flow, check out [its documentation](https://flow.org/).
 
+## Adding ESLint and Prettier
```
Comment (brentvatne) [inline react-native-scripts/template/README.md]: can we split this up so we have an ESLint section and a Prettier section? in CRA the section is called "Formatting Code Automatically" https://github.com/facebookincubator/create-react-app/blob/b555192b43876c423a81f0cd699f3f4a94aa7441/packages/react-scripts/template/README.md#formatting-code-automatically

File: react-native-scripts/template/README.md [hunk 4]
```diff
@@ -142,6 +143,34 @@ You can optionally use a [plugin for your IDE or editor](https://flow.org/en/doc
...
+
+[ESLint](http://eslint.org/) is a configurable JavaScript linter. ESLint can help standardize a style guide across the team.
+
+To add eslint to a Create React Native App project, follow these steps:
+
+1. Run `npm install --save-dev eslint eslint-config-react-native`
+2. Add `"lint": "eslint ."` to the `scripts` section of your `package.json`.
+3. Add the following to the `.eslintrc` file in the project root which is used to configure how eslint should run,
```
Comment (brentvatne) [inline react-native-scripts/template/README.md]: ESLint

File: react-native-scripts/template/README.md (hunk 1 above)
Comment (brentvatne) [inline react-native-scripts/template/README.md]: ESLint

File: react-native-scripts/template/README.md [hunk 5]
```diff
@@ -142,6 +143,34 @@ You can optionally use a [plugin for your IDE or editor](https://flow.org/en/doc
...
+
+```json
+{
+  "extends": ["react-native"]
+}
+```
+
+Please see the repo of [eslint-config-react-native](https://github.com/kunalgolani/eslint-config/tree/master/react-native) to know what rules are enabled with this configuration.
```
Comment (brentvatne) [inline react-native-scripts/template/README.md]: Refer to the [eslint-config-react-native repoistory]([eslint-config-react-native](https://github.com/kunalgolani/eslint-config/tree/master/react-native) to know what rules are enabled with this configuration.) to see the exact rules that are enabled with this configuration.

**Comment by anp** [comment]
Hi! Just doing a little cleanup. If you're still interested in pursuing this PR, please reply back and we can reopen. Thanks!

**Comment by flyingsky** [comment]
This PR is very helpful, do we have plan to merge it?

**Pull request closed without merging**